src/test/java/com/thevoxelbox/voxelsniper/BrushesTest.java

```diff
@@ -47,9 +41,11 @@ public void testGetBrushForHandle() {
     @Test
     public void testRegisteredSniperBrushes() {
         Assert.assertEquals(0, brushes.registeredSniperBrushes());
+        Assert.assertEquals(0, brushes.registeredSniperBrushHandles());
         IBrush brush = Mockito.mock(IBrush.class);
         brushes.registerSniperBrush(brush.getClass(), "mockhandle", "testhandle");
         Assert.assertEquals(1, brushes.registeredSniperBrushes());
+        Assert.assertEquals(2, brushes.registeredSniperBrushHandles());
```

Comment:
From what I see these tests don't quite follow the Triple A principle (Arrange, act, assert). I think we should reorder these tests as closely as possible to that for increased clarity
```suggestion
        IBrush brush = Mockito.mock(IBrush.class);
        Assert.assertEquals(0, brushes.registeredSniperBrushes());
        Assert.assertEquals(0, brushes.registeredSniperBrushHandles());
        brushes.registerSniperBrush(brush.getClass(), "mockhandle", "testhandle");
        Assert.assertEquals(1, brushes.registeredSniperBrushes());
        Assert.assertEquals(2, brushes.registeredSniperBrushHandles());
```

Reply:
perhabs even split off the empty tests to seperate ones?

Reply:
You've got a point. The main reason I added those empty asserts was to make sure the VoxelBrushManager (brushes) is re-initialized before every test. But I am pretty sure this is always re-initialized before a test is ran, so perhaps should we just remove those lines again?